metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/0.3.0/configuration/metron-env.xml

```diff
@@ -246,7 +246,7 @@ bolt.hdfs.file.system.url={{ default_fs }}
 bolt.hdfs.wip.file.path=/paloalto/wip
 bolt.hdfs.finished.file.path=/paloalto/rotated
 bolt.hdfs.compression.codec.class=org.apache.hadoop.io.compress.SnappyCodec
-index.hdfs.output={{ metron_apps_enrichment_dir }}
```

Comment:
That is a fair point, since we do expose the metron-env in Ambari. I'm willing to leave it separately, although it does make me wonder if there's a better way to handle configs like that in general.  Having said that, I don't see a good way around it based on how Ambari needs things defined.